failure |= error("%s", err.buf);